templates/promotion/promotion.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable function-call-argument-newline */
+/* eslint-disable max-len */
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, a } from '../../scripts/dom-helpers.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main > .section');
+  const divPromotion = div({ class: 'promotion-section' });
+  const leftSection = doc.querySelector('.left-column');
```

Comment:
The left-column has a section that's adding additional padding.  Need to figure out why the section is being added here, or we can blow it away.

```
leftSection.classList.remove('section')
```


The result of the previous two updates will align the text with the breadcrumb.
![image](https://github.com/user-attachments/assets/632160a7-7213-4fa2-b652-13475b4a37f9)

vs
![image](https://github.com/user-attachments/assets/afec1402-bb9a-4458-a4ce-843ebed1b80f)